### ISSUE TITLE
Crash on gamestart (Null pointer exception)

### ISSUE THREAD

**Issue by MasterEnderman** [codetaylor/dropt#46]
Crash Report: https://paste.dimdev.org/nahaqehawi.mccrash
Athenaeum: 1.15.0
Dropt: 1.11.2
Forge: 14.23.5.2794

Another qestion i would like to ask if there is another way to communicate with you besides your github repos. I mean i found you on twitter and discord, but would it be okay if i pm you there?

Loving your mods. Heavily using dropt and artisan worktables in my current pack. keep up the good work!

**Comment by codetaylor** [comment]
Thank you. :)

Please show me your script, you might have a stray comma in the blocks array maybe...

I'm working on setting up a discord server for topics related to my mods, would you like me to ping you when it's ready?

**Comment by codetaylor** [comment]
Also, are you using ZenScript, JSON, or the DroptAPI?

**Comment by MasterEnderman** [comment]
Well that are some good news! Sure just ping me, when it's ready :)

I'm using a combintation of ZenScript and JSON as I couldn't figure out yet how to use the "harvester" functions in ZenScript, so I copied the example from your documentation site into a json and edited it. I tested my ZenScript before and there wasn't any errors.
Thank you for pointing me towards a missplaced comma. I just checked my json again and found one. (which ist strange, as my json validator didn't throw an error, before i reloaded the jsons ingame. So guess I'm sorry for causing trouble, where it was my fault all along. So guess I can close this issue.
